stage1/init/common/pod.go

```diff
@@ -386,7 +386,7 @@ func findBinPath(p *stage1commontypes.Pod, appName types.ACName, app types.App,
 }
 
 // appToSystemd transforms the provided RuntimeApp+ImageManifest into systemd units
-func appToSystemd(p *stage1commontypes.Pod, ra *schema.RuntimeApp, interactive bool, flavor string, privateUsers string) error {
+func appToSystemd(p *stage1commontypes.Pod, ra *schema.RuntimeApp, interactive bool, flavor string, privateUsers string, rootFsIsReadOnly bool) error {
```

Comment:
we should really think about refactoring this function, it becomes bigger and bigger, maybe in "[functional options](http://dave.cheney.net/2014/10/17/functional-options-for-friendly-apis)" style.